src/dune_rules/install_rules.ml

```diff
@@ -178,7 +197,13 @@ end = struct
           ~paths:(Disallow_external lib_name)
       in
       Path.Set.to_list_map deps ~f:(fun path ->
-          let path = Path.as_in_build_dir_exn path in
+          let path =
+            let local_path =
+              path |> Path.as_in_build_dir_exn |> Path.Build.local
+            in
+            check_runtime_deps_relative_path ~lib_info:info ~loc local_path;
+            Path.as_in_build_dir_exn path
```

Comment:
I think calling `Path.as_in_build_dir_exn` should only be called once.

Reply:
Good point, just fixed that.